### ISSUE TITLE
Update please

### ISSUE THREAD

**Issue by QPLAYid** [mywalkb/DenylistUnmount#6]
increase zygisk footprint

**Comment by mywalkb** [comment]
Have you some suggestions? Some apps stop to work?

**Comment by QPLAYid** [comment]
yes for some applications it doesn't work, I tried Indomaret point' but it didn't work 
https://apkcombo.com/id/indomaret-poinku/mypoin.indomaret.android/
